### PR TITLE
chore: remove unused services

### PR DESCRIPTION
**Pull request by linuxbandit**
ciaohttp and cachet are replaced by statping

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 # Important variables
 BASE_URL=localhost.test
-
 HOSTNAME=
-EMAIL_DOMAIN=
 
 #we wish :(
 SHORTENER_DOMAIN=aeg.ee
 
+EMAIL_DOMAIN=
```
Comment (serge1peshcoff) [inline .env.example:10]: what is this used for btw?

File: .env.example [hunk 2]
```diff
@@ -20,59 +21,24 @@ BUGSNAG_KEY_EVENTS=123
...
-PATH_OMS_MONITOR=oms-monitor/docker/
-PATH_OMS_ELASTIC=oms-elastic/docker/
-PATH_OMS_WIKI=oms-wiki/docker/
-PATH_OMS_SURVEY=oms-survey/docker/
 
 CORE_LOGIN=admin
 CORE_PASSWORD=5ecr3t
 
-SMTP_HOST=smtp.gmail.com
+SMTP_HOST=smtp.gmail.com #!!!!
```
Comment (serge1peshcoff) [inline .env.example]: why this?

File: .env.example [hunk 3]
```diff
@@ -93,14 +59,47 @@ PW_STATPING=5ecr3t
 
 STATPING_TOKEN=
 
-PHP_APP_KEY=YdEgGZtx98WIBOKJky3gBvcnE+1ZW9+40HHSI0p+vWQ=
-RUBY_APP_KEY=YdEgGZtx98WIBOKJky3gBvcnE+1ZW9+40HHSI0p+vWQ=
+SSH_PORT=
+ELK_VERSION=
 
+#possibly remove?
```
Comment (serge1peshcoff) [inline .env.example]: I think we can safely remove it (and wiki at all) and moving wiki plugin to other repo, as it's not the part of oms-docker and is hosted on zeus

File: .env.example (hunk 2 above)
Comment (linuxbandit) [reply .env.example]: I don't remember and i also needed to remove it because it could break the variable

File: .env.example (hunk 3 above)
Comment (linuxbandit) [reply .env.example]: yes but the objective imho is to bring it to us, we also can add automatic backups etc

Comment (linuxbandit) [reply .env.example]: hm but ok i will remove it so it is in the same commit, we can look back here for the code

File: .env.example (hunk 1 above)
Comment (linuxbandit) [reply .env.example:10]: because BASE_URL will not always be our domain (especially now that i am making all the tests with barnabeu)

**Review by serge1peshcoff — APPROVED** [review]
lgtm

**Pull request merged**